lib/store/actions.js

```diff
@@ -242,6 +244,42 @@ const actions = {
       },
     };
   },
+
+  startWatchingLocation() {
+    return (dispatch) => {
+      dispatch(actions.locationWatchingStarted());
+      watchID = navigator.geolocation.watchPosition(
```

Comment:
Should we stop watching location prior doing a start, to avoid having two? Or is a case that wont happen?